test/fail_compilation/retscope2.d

```diff
@@ -162,3 +162,67 @@ void foo800()
     scope Exception e;
     throw e;
 }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
The point of this is that there are 3 PRs, and I'll have to do a rebase for two of them. These blank lines will make that easier. The test cases already use `#line`.